apps/mark-scan/backend/src/custom-paper-handler/application_driver.ts

```diff
@@ -36,6 +37,10 @@ export async function setDefaults(
   debug('set line spacing to 0');
   await driver.setPrintingSpeed('slow');
   debug('set printing speed to slow');
+
+  const scanDirection: ScanDirection = 'backward';
+  await driver.setScanDirection(scanDirection);
+  debug('set scan direction to', scanDirection);
```

Comment:
For https://github.com/votingworks/vxsuite/issues/4812 you may need to move `setScanDirection` into individual states, but you could also leave this default, change the direction for the re-cast flow, and change it back after. Up to you.